src/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,22 @@
+import cn from 'classnames';
+
+export const Tab = ({ tab, selectedTab, onTabSelected }) => (
+  <li
+    className={cn({
+      'is-active': tab.id === selectedTab.id,
+    })}
+    data-cy="Tab"
+  >
+    <a
+      href={`#${tab.id}`}
+      data-cy="TabLink"
+      onClick={() => {
+        if (tab.id !== selectedTab.id) {
+          onTabSelected(tab);
+        }
+      }}
```

Comment:
You can write a function handler for it, not use anonim function ( not critical)